products.html

```diff
@@ -0,0 +1,49 @@
+<!doctype html>
+
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+
+  <title>Sprint Challenge - About</title>
+
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+  <link rel="stylesheet" href="css/index.css">
+
+</head>
+
+<body>
+
+    <nav class ="alpha">
+        <img src="img/lambda-black.png" alt="Lambda Logo">
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="products.html">Products</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+    </nav>
+
+     <div class="container products-page">
+
+    <h1>Products are pending.</h1>     
+    <p>Here are some buttons!</p>
+    <button class="jumpy"><a href ="#">1</a></button>
+    <button class="jumpy"><a href ="#">2</a></button>
+    <button class="jumpy"><a href ="#">3</a></button>
+    <button class="jumpy"><a href ="#">4</a></button>
+    <button class="jumpy"><a href ="#">5</a></button>
+   
+   
+   
+   <footer>
+        <nav>
+          <a href="index.html">Home</a>
+          <a href="#">About</a>
+          <a href="#">Products</a>
+          <a href="#">Blog</a>
+          <a href="#">Contact</a>
+        </nav>
+      </footer>
+   
+    </div>
+    </body>
+</html>
```

Comment:
Awesome job spending the extra time to stretch and build out more on the Sprint.